src/order/bounded_lattice.lean

```diff
@@ -26,6 +26,8 @@ class has_bot (α : Type u) := (bot : α)
 notation `⊤` := has_top.top _
 notation `⊥` := has_bot.bot _
 
+attribute [pattern] lattice.has_bot.bot lattice.has_top.top
```

Comment:
What does this do?

Reply:
It allows us to write `⊤` and `⊥` in the equation compiler, like this:
```lean
protected def neg : ereal → ereal
| ⊥       := ⊤
| ⊤       := ⊥
| (x : ℝ) := (-x : ℝ)
```